### DISCUSSION TITLE
Foobar2000 integration perhaps?

### DISCUSSION THREAD

**Discussion by tomasts248** [navidrome/navidrome#2327] [Ideas]
Foobar already has a desktop player and a mobile app, so it 'd be very useful.

**Comment by dweymouth** [comment]
There was a Foobar2000 [subsonic plugin](https://github.com/hypfvieh/foo_subsonic) but it's no longer maintained. You may want to look into using dedicated Subsonic clients like [Supersonic](https://github.com/dweymouth/supersonic) on desktop (disclosure: I'm the dev) and something like DSub or Ultrasonic or Symfonium on Android; or iSub/play:sub/Amperfy on iOS. See here [https://github.com/basings/selfhosted-music-overview](https://github.com/basings/selfhosted-music-overview) for an overview of many of the clients.

**Reply by tomasts248** [reply]
Thank you!! 

> There was a Foobar2000 [subsonic plugin](https://github.com/hypfvieh/foo_subsonic) but it's no longer maintained.

Okey I didnt explain well sorry: I would like to use Foobar as the source for Navidrome so that I can access my Foobar playlists, and access them with a browser to play these playlists remotely, that would do.

> https://github.com/basings/selfhosted-music-overview

Woa thats a lot of info to process... May I ask if I need an open source and free that supports collaboration. Also is subsonic open source and free?

**Reply by deluan** [reply]
> Okey I didnt explain well sorry: I would like to use Foobar as the source for Navidrome so that I can access my Foobar playlists, and access them with a browser to play these playlists remotely, that would do.

Navidrome can auto-import any playlists found in the MusicFolder. You can configure Foobar to save the playlists in a subdirectory under MusicFolder and the playlists will be visible by Navidrome.

> Also is subsonic open source and free?

[Subsonic the product](http://subsonic.org) was Open Source but it is not anymore. It has a free tier, but it is not maintained anymore so you may want to use a different solution.

[Subsonic the API](http://www.subsonic.org/pages/api.jsp) was implemented by different servers and clients, and it is [alive and kicking](https://opensubsonic.netlify.app/), with a thriving ecosystem of servers and clients available

**Reply by dweymouth** [reply]
Does Navidrome recognize Foobar's .fpl playlist files? They are just plain text files but with a non-standard extension (and format)?

**Reply by deluan** [reply]
Ah, I never used it, and assumed it was saving the playlists as .m3u files. 

By the way, a quick search says it is not a "plain text file": https://hydrogenaud.io/index.php/topic,43673.0.html

**Reply by dweymouth** [reply]
Good catch. I think I'm confusing Foobar's custom playlist files with its custom equalizer curve files which were plaintext. It's been ages since I used Foobar. I wonder if someone has implemented a .fpl to .m3u converter

Edit: @tomasts248  you might be able to do it somewhat manually: https://hydrogenaud.io/index.php/topic,19471.0.html

**Reply by tomasts248** [reply]
@dweymouth Yeah I do that from time to time (for backups) but is cumbersome since I edit it very often. I'm also having "auto-playlists". 

> I wonder if someone has implemented a .fpl to .m3u converter

I wonder too since Foobar already converts them manually, maybe there's a cli or run parameter for converting playlist and could be set to update every x minutes?
